File: Basics/C#/CSharp.Basics/Samples/StaticObjects/AdoptionHelper.cs
using CSharp.Basics.Samples.Generics;
using System;
using System.Collections.Generic;
using System.Text;

namespace CSharp.Basics.Samples.StaticObjects
{
    public static class AdoptionHelper
    {
        public static void AdoptPet(AdoptablePet pet)
        {
            pet.IsAdopted = true;
        }
    }
}

File: Basics/Kotlin/app/src/main/java/com/suavepirate/kotlinbasics/samples/staticObjects/AdoptionHelper.kt
package com.suavepirate.kotlinbasics.samples.staticObjects

import com.suavepirate.kotlinbasics.samples.generics.AdoptablePet

object AdoptionHelper {
    fun adoptPet(pet: AdoptablePet) {
        pet.isAdopted = true;
    }
}

fun run() {
    AdoptionHelper.adoptPet(AdoptablePet(false))
}